css/BEM-style.md

```diff
@@ -156,22 +156,22 @@ You can now pretty much style all the components the way you like, just by using
     text-decoration: none;
     border:1px solid saddlebrown;
     padding:1rem 2rem;
-    color:white;
+    color: white;
```

Comment:
If you want to add white space like this, do it everywhere as I see attributes where you not did the same change.